node/src/components/consensus/highway_core/highway.rs

```diff
@@ -245,6 +246,7 @@ impl<C: Context> Highway<C> {
         let instance_id = self.instance_id.clone();
         self.map_active_validator(
             |av, state, rng| av.handle_timer(timestamp, state, instance_id, rng),
+            timestamp,
```

Comment:
@afck Here in `handle_timer` we just use the timer's `timestamp`, and assume its ~ `Timestamp::now()`

Reply:
I have added a comment explaining this.